src/platform_impl/windows/event_loop.rs

```diff
@@ -2228,3 +2091,183 @@ unsafe extern "system" fn thread_event_target_callback<T: 'static>(
     }
     result
 }
+
+unsafe fn handle_raw_input<T: 'static>(
+    subclass_input: &Box<ThreadMsgTargetSubclassInput<T>>,
+    data: RAWINPUT,
+) {
+    use crate::event::{
+        DeviceEvent::{Button, Key, Motion, MouseMotion, MouseWheel},
+        ElementState::{Pressed, Released},
+        MouseScrollDelta::LineDelta,
+    };
+
+    let device_id = wrap_device_id(data.header.hDevice as _);
+
+    if data.header.dwType == winuser::RIM_TYPEMOUSE {
+        let mouse = data.data.mouse();
+
+        if util::has_flag(mouse.usFlags, winuser::MOUSE_MOVE_RELATIVE) {
+            let x = mouse.lLastX as f64;
+            let y = mouse.lLastY as f64;
+
+            if x != 0.0 {
+                subclass_input.send_event(Event::DeviceEvent {
+                    device_id,
+                    event: Motion { axis: 0, value: x },
+                });
+            }
+
+            if y != 0.0 {
+                subclass_input.send_event(Event::DeviceEvent {
+                    device_id,
+                    event: Motion { axis: 1, value: y },
+                });
+            }
+
+            if x != 0.0 || y != 0.0 {
+                subclass_input.send_event(Event::DeviceEvent {
+                    device_id,
+                    event: MouseMotion { delta: (x, y) },
+                });
+            }
+        }
+
+        if util::has_flag(mouse.usButtonFlags, winuser::RI_MOUSE_WHEEL) {
+            let delta = mouse.usButtonData as SHORT / winuser::WHEEL_DELTA;
```

Comment:
this reverts: https://github.com/rust-windowing/winit/commit/86748fbc681ea070bc37e90a569c192555b402cd#diff-bb05c5e1236bdaa7f268bcbbbb8cc9f378a48eb3ed480884f0c6471a9cc933edL2104

Reply:
Well spotted! I'm fixing this right away